src/helpers/cloudnary.js

```diff
@@ -0,0 +1,10 @@
+import dotenv from 'dotenv';
+import cloudinary from 'cloudinary';
+
+dotenv.config();
+
+cloudinary.config({
```

Comment:
can you please put this as the CLOUDINARY_URL in .env?? so that we may have the same structure

Reply:
It was conflicting with your Url .and i could not continue uploading using your account .I was getting access denied error